ansible/roles/telegraf/meta/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Copyright (C) 2021 Dr. Serge Victor <https://dr.sergevictor.eu/>
```

Comment:
You should add this line to all files created in this PR.

Reply:
Done, thanks for pointing it out.